site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -27,7 +27,7 @@ Install Postman from [the Postman downloads site :material-dock-window:](https:/
 
 Once Postman is installed, start up the application.  You should see an initial page something like this:
 
-![Postman main window](postman-mainmenu.png)
+![Postman main window](postman-main-menu.png)
```

Comment:
I suggest we should mention what platform and version of postman you have taken the screen shots. e.g. on a Mac M1 postman version ... this should match the postman version mentioned earlier.

Reply:
I've added this prior to displaying the first screenshot.